extensions/jaeger/runtime/pom.xml

```diff
@@ -34,6 +34,10 @@
             <groupId>io.smallrye</groupId>
             <artifactId>smallrye-metrics</artifactId>
         </dependency>
+        <dependency>
```

Comment:
before someone asks, this is needed because `microprofile-metrics-api` is now a `provided` dependency of SR Metrics so it's not brought in transitively